Command/GenerateAdminCommand.php

```diff
@@ -53,7 +53,7 @@ public function configure()
             ->addOption('admin', 'a', InputOption::VALUE_OPTIONAL, 'The admin class basename')
             ->addOption('controller', 'c', InputOption::VALUE_OPTIONAL, 'The controller class basename')
             ->addOption('manager', 'm', InputOption::VALUE_OPTIONAL, 'The model manager type')
-            ->addOption('services', 'y', InputOption::VALUE_OPTIONAL, 'The services YAML file', 'services.yml')
+            ->addOption('services', 'y', InputOption::VALUE_OPTIONAL, 'The services YAML file', null)
```

Comment:
It does look weird. IMO the changed code should be in `interact()`, this for when you use the command in non-interactive mode, by passing options.